extras/rya.console/src/main/java/mvm/rya/shell/RyaAdminCommands.java

```diff
@@ -108,6 +127,9 @@ public boolean arePCJCommandsAvailable() {
         final ShellState shellState = state.getShellState();
         if(shellState.getConnectionState() == ConnectionState.CONNECTED_TO_INSTANCE) {
             final GetInstanceDetails getInstanceDetails = shellState.getConnectedCommands().get().getGetInstanceDetails();
+            if(getInstanceDetails instanceof MongoGetInstanceDetails) {
+                return false;
+            }
```

Comment:
This is kind of a hack. Store what type of Rya storage we are connected to within the shared shell state and then key off of that.